.github/workflows/website_frontend_ci.yml

```
@@ -10,14 +10,15 @@ env:
   AZURE_WEBAPP_NAME: Jordnaer
   RESOURCE_GROUP: Jordnaer
+  PLAYWRIGHT_VERSION: 1.54.0
 
 jobs:
   playwright:
     # Only run this if the workflow that triggered it was successful
     if: ${{ github.event.workflow_run.conclusion == 'success' }}
     runs-on: ubuntu-latest
     container:
-      image: mcr.microsoft.com/playwright/dotnet:v1.50.0-jammy
+      image: mcr.microsoft.com/playwright/dotnet:v${{ env.PLAYWRIGHT_VERSION }}-jammy
```

actionlint (1.7.7)

context "env" is not allowed here. available contexts are "github", "inputs", "matrix", "needs", "strategy", "vars". see https://docs.github.com/en/actions/learn-github-actions/contexts#context-availability for more details

(expression)